src/Tribe/Views/V2/View.php

```diff
@@ -1778,7 +1778,7 @@ public function get_title( array $events = [] ) {
 		 */
 		$title = apply_filters( "tribe_events_views_v2_view_{$slug}_title", $title, $this );
 
-		return htmlspecialchars_decode($title);
+		return html_entity_decode($title, ENT_QUOTES);
```

Comment:
Space inside parentheses 

```suggestion
		return html_entity_decode( $title, ENT_QUOTES );
```